src/mono/mono/mini/aot-compiler.c

```diff
@@ -8522,6 +8525,8 @@ mono_aot_parse_options (const char *aot_options, MonoAotOptions *opts)
 			opts->profile_files = g_list_append (opts->profile_files, g_strdup (arg + strlen ("profile=")));
 		} else if (!strcmp (arg, "profile-only")) {
 			opts->profile_only = TRUE;
+		} else if (str_begins_with (arg, "mibc-profile=")) {
+			opts->mibc_profile_files = g_list_append (opts->mibc_profile_files, g_strdup (arg + strlen ("mibc-profile=")));
```

Comment:
Do we want to keep the old aot profiler code ? If not, this should be named "profile=".

Reply:
Yes, I think we should keep it, but treat it as deprecated.

Reply:
SInce xamarin-android is using .aotprofile, I think we should maintain this option. And while keeping both, perhaps there can be comparisons between profiling with .mibc and .aotprofile. When the full dotnet-pgo based profiled AOT workflow is finished, and when all mono Profiled AOT users are aware of the changes to the pipeline, then maybe we can remove the old code?

Reply:
Yes, since generating aot profiles is important for Android, we would need to make sure the new format is acceptable before removing the old.